packages/shared/ReactTypes.js

```diff
@@ -66,6 +66,9 @@ export type ReactContext<T> = {
   // DEV only
   _currentRenderer?: Object | null,
   _currentRenderer2?: Object | null,
+  // This value may be added by application code
+  // to improve DEV tooling display names
+  displayName?: string,
```

Comment:
This helps me avoid a bunch of `: any` casts for DevTools code and matches the [Flow definition](https://github.com/facebook/flow/blob/416d5bc1d75cf70acd81b4cd29b713581883fb6a/lib/react.js#L211-L212).

Reply:
And would help #18035 :wink: